docs/06-01-Fine-tuning-view.md

```diff
@@ -118,7 +119,7 @@ For text classification, accuracy is often not a very good measure. If datasets
 
 #### How do you measure the score?
 
-When training our machine learning model, we leave out a part of the data you have reviewed. Scoring is then done on the left-out data.
+Under the hood, the model does not only assign topics to the rows, but it also forcast a certainty measure for those predictions. We can estimate the AI score starting from such certaintly measures.
```

Comment:
@marco-caplena  typo "certaintly" "certainty"

Reply:
certainty is correct, right?